modules/verifying-image-signatures-to-ensure-image-integrity.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * operating/verify-image-signatures.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="verifying-image-signatures-to-ensure-image-integrity_{context}"]
+= Verifying image signatures to ensure image integrity
+
+By scanning an image signature, you can verify that a trusted source signed the container image and the scan output includes a valid transparency log bundle to ensure image integrity.
```

Comment:
I think we should mention a tricky edge case here. When the image in question is a multi-arch image, one has to be extra careful with signing the "correct" digest. That is because multi-arch images are presented by an index image (identified by the index digest), that acts a container for the architecture specific manifests (e.g. `amd64` and `arm64`). The latter then make up the architecture specific images (each identified by the architecture digest). Now in this case, we recommend signing both the "outer" index digest and all the "inner" architecture specific digests. This avoid problems with images resolution by the container runtime.